dashboard/app/helpers/levels_helper.rb

```diff
@@ -215,6 +215,17 @@ def app_options
       shouldShowDialog: @level.properties['skip_dialog'].blank? && @level.properties['options'].try(:[], 'skip_dialog').blank?
     }
 
+    if current_user
+      section = current_user.sections_as_student.find_by(script_id: @script.try(:id)) ||
+        current_user.sections_as_student.first
+      @app_options[:experiments] =
+        Experiment.get_all_enabled(user: current_user, section: section).map(&:name)
+      if section && section.first_activity_time.nil?
+        section.first_activity_time = DateTime.now
```

Comment:
This could be one-lined as `section.update(first_activity_time: DateTime.now)`. Is the lack of a `!` intentional?